src/ngTouch/directive/ngClick.js

```diff
@@ -238,6 +244,8 @@ ngTouch.directive('ngClick', ['$parse', '$timeout', '$rootElement',
     element.on('touchend', function(event) {
       var diff = Date.now() - startTime;
 
+      // Use JQuery originalEvent
+      event = event.originalEvent || event;
       var touches = (event.changedTouches && event.changedTouches.length) ? event.changedTouches :
           ((event.touches && event.touches.length) ? event.touches : [event]);
       var e = touches[0].originalEvent || touches[0];
```

Comment:
Just a suggestion: as far as I know, there's no "originalEvent" in `touches[0]` - only in the event itself. This would allow you to reduce these two variables to:

```javascript
var e = (event.changedTouches && event.changedTouches.length) ? event.changedTouches[0] :
    ((event.touches && event.touches.length) ? event.touches[0] : event;
```